### PR TITLE
README.rst: Added CRUD examples as mini-documentation

### PR DESCRIPTION
**Pull request by MartinThoma**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -71,6 +71,84 @@ Python versions::
 
     $ tox
 
+
+CRUD
+----
```
Comment (methane) [inline README.rst]: Could you rename this section to "CRUD Exapmle"?

File: README.rst [hunk 2]
```diff
@@ -71,6 +71,84 @@ Python versions::
...
+.. code:: python
+
+    import pymysql
+    connection = pymysql.connect(host='host',
+                                 user='user',
+                                 passwd='passwd',
+                                 db='db')
+    with closing(connection.cursor()) as cursor:
+        sql = ("CREATE TABLE `users` ("
+               "`id` int(11) NOT NULL, "
```
Comment (methane) [inline README.rst]: As far as I know, "C" in "CRUD" is "Creating new record". It means "INSERT", not "CREATE TABLE".

File: README.rst [hunk 3]
```diff
@@ -71,6 +71,84 @@ Python versions::
...
+    import pymysql
+    connection = pymysql.connect(host='host',
+                                 user='user',
+                                 passwd='passwd',
+                                 db='db',
+                                 cursorclass=pymysql.cursors.DictCursor)
+    cursor = connection.cursor()
+
+    sql = ("SELECT `id`, `password` "
+           "FROM `users` WHERE `email`=%s") % email
```
Comment (methane) [inline README.rst]: Don't use %

File: README.rst (hunk 2 above)
Comment (MartinThoma) [inline README.rst]: Oh, sure, you are right. I'll fix that.

File: README.rst [hunk 4]
```diff
@@ -71,6 +71,84 @@ Python versions::
...
+
+.. code:: python
+
+    import pymysql
+    connection = pymysql.connect(host='host',
+                                 user='user',
+                                 passwd='passwd',
+                                 db='db',
+                                 cursorclass=pymysql.cursors.DictCursor)
+    cursor = connection.cursor()
```
Comment (methane) [inline README.rst]: `with connection.cursor() as cursor:`
to ensure closing cursor.

File: README.rst [hunk 5]
```diff
@@ -71,6 +71,84 @@ Python versions::
...
+
+    sql = ("SELECT `id`, `password` "
+           "FROM `users` WHERE `email`=%s") % email
+    cursor.execute(sql)
+    result = cursor.fetchone()
+    connection.close()
```
Comment (methane) [inline README.rst]: Use try~finally to close connection.

File: README.rst (hunk 3 above)
Comment (MartinThoma) [inline README.rst]: What should I use instead?

Comment (methane) [inline README.rst]: See https://www.python.org/dev/peps/pep-0249/#id15

**Comment by MartinThoma** [comment]
I think I have added all suggested changes.

File: README.rst [hunk 6]
```diff
@@ -71,6 +71,118 @@ Python versions::
...
+                                 user='user',
+                                 passwd='passwd',
+                                 db='db',
+                                 cursorclass=pymysql.cursors.DictCursor)
+    cursor = connection.cursor()
+
+    sql = ("SELECT `id`, `password` "
+           "FROM `users` WHERE `email`=%s")
+
+    try:
```
Comment (methane) [inline README.rst]: try should be right after `pymysql.connect`.

File: README.rst [hunk 7]
```diff
@@ -71,6 +71,118 @@ Python versions::
...
+    import pymysql
+    connection = pymysql.connect(host='localhost',
+                                 user='user',
+                                 passwd='passwd',
+                                 db='db',
+                                 cursorclass=pymysql.cursors.DictCursor)
+    cursor = connection.cursor()
+
+    sql = ("SELECT `id`, `password` "
+           "FROM `users` WHERE `email`=%s")
```
Comment (methane) [inline README.rst]: I prefer single line for short query.

**Comment by methane** [comment]
I feel writing all CRUD is too verbose to describe how to use PyMySQL.
Could you remove U and D, and combine all other to single example script?

**Comment by MartinThoma** [comment]
@methane Ok, I've adjusted the example to a single example script.

**Comment by MartinThoma** [comment]
@methane Why don't you merge it?

**Comment by methane** [comment]
Since I have job, I'm maintainer of some OSS projects other than this, and I have many OSS projects to contribute.
Reviewing pull request requires some time and a lot of mental power.
Please don't put pressure for issues other than vulnerability or critical regression.

**Pull request merged**